docs/src/pages/[platform]/connected-components/authenticator/customization/customization.styling.react-native.mdx

```diff
@@ -0,0 +1,20 @@
+
+## Styling
+
+### Theme Provider
+
+You can update the style of the Authenticator by wrapping it with a [ThemeProvider](/react-native/theming). Then create a [theme object](/react-native/theming#theme-structure), with all your font customizations.
+
+```jsx file=../../../../../../../examples/react-native/src/features/Authenticator/Theming/Example.tsx
```

Comment:
I assume this is a good idea so we can test the example even if we aren't rendering it in the docs